src/mon/Monitor.cc

```diff
@@ -3607,6 +3671,11 @@ void Monitor::handle_command(MonOpRequestRef op)
     f->flush(rdata);
     rs = "";
     r = 0;
+  } else if (prefix == "mon status") {
+    print_mon_status(ds);
```

Comment:
why not pass `f` formatter to `print_mon_status()` instead?

Reply:
I think `ostream` is enough for printing out information to the users, so formatter does not need to be passed.

Reply:
i see. but how a script is supposed to parse the output?